### ISSUE TITLE
UseHttps effects not documented

### ISSUE THREAD

**Issue by tomten** [umbraco/UmbracoDocs#3123]
## What type of issue is it? (Choose one - delete the others)
Missing documentation

## What article/section is this about?
Reference > Config

## Describe the issue
The effects of `UseHttps` could be documented to 
- clarify its intended use & 
- increase the adoption rate. 

To my understanding, from searching the forums, the only effect of this configuration setting is that the Admin auth cookie is marked as https-only, with the side effect of disabling non-https-sign-in. But I'd like to know for sure. :)

The current documentation says to configure this "when the site is running on https", which is a bit ambiguous. 

For example, when running on Azure, there are _at least_ two other ways of https-onlying Admin:
- URL rewrite configuration to upgrade to https
- Azure "https only" setting for the app service

Mentioning how this relates to, if it does, `UseHttps` could also be useful.

**Comment by sofietoft** [comment]
Thanks for the very detailed report here @tomten !

As I understand it, you're looking for some more details and clarification as to what it means to use the `useHttps` setting.
I've found that the setting is documented in various places in the docs - not entirely sure where you've been looking? 🤔 

In any case, I've found info about the setting in these places:
- [Extending > Health Check](https://our.umbraco.com/documentation/Extending/Health-Check/)
- [Reference > Web.config](https://our.umbraco.com/documentation/Reference/Config/webconfig/#umbracocoreusehttps)
- [Reference > Security > Use Https](https://our.umbraco.com/documentation/Reference/Security/use-https)

If you haven't read those articles already - perhaps the info you're looking for, can be found there?
Let me know, and I'll of course make sure that we look into adding some more details on the setting, if you think it's missing 💪

**Comment by tomten** [comment]
Reference > Security > Use Https does indeed answer all my questions. So I guess my real problem was that that page does not appear in this search:
 
https://our.umbraco.com/search?q=usehttps&cat=documentation&v=8

**Comment by sofietoft** [comment]
Aha! 🤔 That's also a very valid problem! Something like this should be possible to find, for sure!

What keywords did you use to search?
And did you search on Our, or through Google?

**Comment by tomten** [comment]
> What keywords did you use to search?

The keyword is in the link. :)

> And did you search on Our, or through Google?

I searched on Our.

**Comment by sofietoft** [comment]
Right! I completely missed that you added a link to the search 😅 

In any case, I've updated the HTTPs article now. Added some meta data (title and description) and made sure that "UseHttps" is used in the actual text on the page as well.
Looks like it's showing up in the search results now: https://our.umbraco.com/search?q=usehttps&cat=documentation&v=8 

Thanks once again for making us aware of this issue! 💪
